region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -1051,6 +1052,33 @@
         "At the beginning of the climb, avoid pressing left without X-Ray being held, to prevent triggering the transition of the bottom door."
       ]
     },
+    {
+      "link": [4, 2],
+      "name": "Wall Ice Clip X-Ray Climb",
+      "requires": [
+        {"notable": "Wall Ice Clip X-Ray Climb"},
+        "canWallIceClip",
+        {"thornHits": 1},
```

Comment:
I often lose iframes once while standing solidly inside the thorns, in a spot that doesn't deal damage.
![Screenshot 2025-01-04 072005](https://github.com/user-attachments/assets/a81664d0-a07a-40ca-ba57-178f07d87ade)
Then I take an unavoidable 2nd hit when I crouch.

Reply:
Interesting, I don't know exactly what's going but was able to reproduce it now. It never happened to me before, apparently because after starting to climb I always crouched before pressing X-Ray again. Added a line in the notes to say to do this.